Gemfile

```diff
@@ -42,7 +42,8 @@ gem 'slim-rails'
 gem 'split', require: 'split/dashboard'
 gem 'staccato'
 gem 'twilio-ruby'
-gem 'two_factor_authentication', github: 'Houdini/two_factor_authentication'
+gem 'two_factor_authentication', github: 'amoose/two_factor_authentication',
+                                 branch: 'feature/more_options'
```

Comment:
If we like this change can you make a PR upstream?  And add a comment here about why we are not pointing at upstream?  (And include a like to the PR so we no when we can un-fork).